### ISSUE TITLE
Feature Request: Backend connection to network share.

### ISSUE THREAD

**Issue by ithakaa** [projectsend/projectsend#960]
It would be wise to allow a backend connection to a file server which would then make the process of sharing files much easier than uploading to projectsend.

Use case:

> Projectsend backend to connect to a network share hosting company files.
> Network administrator selects backend file(s) and makes them available to client via projectsend frontend.

In this case, there is no need for the administrator to upload files to projectsend

**Comment by ignacionelson** [comment]
I don't think this will be easy to integrate in the current codebase but I'll keep it in mind. Many things are hardcoded today and these kind of features requires a full rewrite (which I've been slowly working on but it's still very much on it's infancy right now).

**Comment by ignacionelson** [comment]
This issue is being closed as part of a bulk cleanup. ProjectSend has undergone significant changes since this was reported (including r1720 and r1994), and many older issues no longer apply to the current codebase.

If you are still experiencing this problem on the latest version, please open a new issue with updated details and we'll be happy to look into it. Thank you for your contribution!
